src/scanner/rule-to-links-mappings.ts

```diff
@@ -17,6 +17,13 @@ const BestPractice: HyperlinkDefinition = {
 };
 
 export const ruleToLinkConfiguration: DictionaryStringTo<HyperlinkDefinition[]> = {
+    'aria-input-field-name': [link.WCAG_4_1_2],
+    'aria-toggle-field-name': [link.WCAG_4_1_2],
+    'avoid-inline-spacing': [link.WCAG_1_4_12],
+    'input-button-name': [link.WCAG_4_1_2],
+    'landmark-unique': [BestPractice],
+    'role-img-alt': [link.WCAG_1_1_1],
+    'scrollable-region-focusable': [BestPractice],
```

Comment:
`scrollable-region-focusable` should actually be mapped to 2_1_1, but it seems like mapping to `BestPractice` is how we exclude rules from being run, regardless of their original designation from `axe`. Am I missing something?

Reply:
You're correct. I'd recommend mapping to both for clarity (this will still exclude them), similar to how `landmark-complementary-is-top-level` is marked up below

Reply:
Good idea, updated

Reply:
this rule had a false positive, we dont want to enable it until we consume the fix too